config.xsd

```diff
@@ -452,6 +452,7 @@
             <xs:element name="UnusedClass" type="ClassIssueHandlerType" minOccurs="0" />
             <xs:element name="UnusedClosureParam" type="IssueHandlerType" minOccurs="0" />
             <xs:element name="UnusedConstructor" type="MethodIssueHandlerType" minOccurs="0" />
+            <xs:element name="UnusedForeachValue" type="IssueHandlerType" minOccurs="0"></xs:element>
```

Comment:
maybe keep the same  convention of self closing tags

Reply:
oops